frontend/www/js/omegaup/components/arena/NavbarMiniranking.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <table class="mini-ranking" v-if="showRanking">
+    <thead>
+      <tr>
+        <th></th>
+        <th>{{ T.wordsUser }}</th>
+        <th class="total" colspan="2">{{ T.wordsTotal }}</th>
+        <th></th>
+      </tr>
+    </thead>
+    <tbody class="user-list-template">
+      <tr v-for="user in users">
+        <td class="position">{{ user.position }}</td>
+        <td class="user">
+          <omegaup-user-username
+            v-bind:classname="user.classname"
+            v-bind:username="user.username"
+            v-bind:country="user.country"
+          >
+          </omegaup-user-username>
+        </td>
+        <td class="points">
+          {{ user.points.toFixed(digitsAfterDecimalPoint) }}
```

Comment:
```suggestion
          {{ user.points.toFixed(2) }}
```

como no le estás pasando este parámetro en el único lugar donde lo estás mandando llamar, mejor hardcodéalo.

Reply:
Listo